core/db.py

```
@@ -6,50 +6,52 @@
 
 from __future__ import annotations
 
+import importlib
+import logging
 import os
 from contextlib import asynccontextmanager, contextmanager
+from types import ModuleType
 from typing import Any, AsyncGenerator, Generator, Optional, TYPE_CHECKING, cast
```

Flake8 (7.3.0)

[error] 'typing.cast' imported but unused

(F401)